pkgs/data/fonts/nerdfonts/default.nix

```diff
@@ -1,26 +1,50 @@
-{ stdenv, fetchFromGitHub, which, withFont ? "" }:
+{ lib, stdenv, fetchFromGitHub
+, which, fonts ? []
+, patchFontDrvs ? [], patchFlags ? [ "mono" "windows" "complete" "careful" ], fontforge ? null }:
```

Comment:
Is this a list of fonts that will be patched and added to the output of nerdfonts?
I think this warrants a section in the [packages](https://nixos.org/nixpkgs/manual/#chap-packages) chapter of the manual.

Reply:
`fonts` has the same purpose of `withFont` but `patchFontDrvs` is meant for patching the fonts through packages provided through Nix, i.e. `pkgs.anonymousPro`. 

The reason the PR is a draft is also because of this, I'm not sure how packages that reference other dependencies should be treated.